features/src/main/feature/camel-features.xml

```diff
@@ -351,12 +351,13 @@
         <bundle>wrap:mvn:software.amazon.awssdk/ses/2.24.0</bundle>
         <bundle>mvn:org.apache.camel.karaf/camel-aws2-ses/${project.version}</bundle>
     </feature>
-<!--    <feature name='camel-aws2-sns' version='${project.version}' start-level='50'>-->
-<!--        <feature version='${project.version}'>camel-core</feature>-->
-<!--        <bundle dependency='true'>mvn:commons-io/commons-io/${commons-io-version}</bundle>-->
-<!--        <bundle dependency='true'>mvn:org.apache.servicemix.bundles/org.apache.servicemix.bundles.aws-java-sdk2/${aws-sdk-v2-bundle-version}</bundle>-->
-<!--        <bundle>mvn:org.apache.camel/camel-aws2-sns/${project.version}</bundle>-->
-<!--    </feature>-->
+    <feature name='camel-aws2-sns' version='${project.version}' start-level='50'>
+        <feature version='${project.version}'>camel-core</feature>
+        <feature>awssdk</feature>
+        <bundle dependency='true'>mvn:commons-io/commons-io/2.14.0</bundle>
```

Comment:
This is not the expected version of commons-io according to the Camel repository https://github.com/apache/camel/blob/camel-4.4.x/parent/pom.xml#L105

Reply:
ok, updated to 2.15.1